wfl/Profiles/RootProfile.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Hypermedia.Affordances;
+using Hypermedia.Transforms;
+using Hypermedia;
+using Symphono.Wfl.Controllers;
+using Symphono.Wfl.Models;
+
+namespace Symphono.Wfl.Profiles
+{
+    public class RootProfile: EntityProfile<Root>
+    {
+        protected override void Configure(EntityConfiguration<Root> configuration)
+        {
+            configuration
+                .UseRepresentationTransform(representation => representation
+                    .WithRepresentation("root")
+                    .WithRepresentation("collection")
+                )
+                .UseLinkTransform(links => links
+                    .WithLink(l => l
+                        .WithRelation("self")
+                        .WithRepresentation("root")
+                        .WithLink<Root, RootController>(
+                            root => rc => rc.Get()
+                        )
+                    )
+               )
+               .UseLinkTransform(links => links
+                    .WithLink(l => l
+                        .WithRelation("restaurants")
+                        .WithRepresentation("restaurant")
```

Comment:
A relation describes how a given item relates to the item it is on.  A representation is API defined and can be used to inform a consumer regarding expected API behavior (class property in Siren).  In this case I think it makes sense to represent the type of class to expect when you follow the link.  We could also choose to just leave this class off as well though.